library/templates/dcp_cpdb.yml

```diff
@@ -0,0 +1,38 @@
+dataset:
+  name: &name dcp_cpdb
+  version: "{{ version }}"
+  acl: private 
```

Comment:
@damonmcc I saw your commend on the nycoc comment that it should be public, I'm assuming this one should be public as well, right?

Reply:
@fvankrieken @damonmcc I think we do want them to be public but so far we've been having trouble dumping the csv to digital ocean for the nycoc_checkbook data when we changed it from private to public - so, might leave this as private for the time being if we can't fix that soon